MfciPkg/Docs/Mfci_Integration_Guide.md

```diff
@@ -56,6 +57,13 @@ must be overridden. Two other instances are available in the MfciPkg, or a custo
   MfciRetrievePolicyLib|MfciPkg/Library/MfciRetrievePolicyLibViaVariable/MfciRetrievePolicyLibViaVariable.inf
 ```
 
+To ensure that all Mfci related variables are locked, the MuVarPolicyFoundationDxe driver needs to be added as well.

```

Comment:
This is an implementation detail of variable policy.

Is there a section in variable policy documentation that mentions this that you can link to instead?

Reply:
Updated to point to the MuVarPolicyFoundationDxe driver documentation.